app/app.go

```diff
@@ -492,6 +492,7 @@ func New(
 		&app.AccountKeeper,
 		&app.UIBCTransferKeeper.Keeper,
 		&app.bech32IbcKeeper,
+		sdk.MustAccAddressFromBech32("umee1uuwjqrgyphm4ac20dufs7dyz0rjl3un49jg8xe"),
```

Comment:
ideally, the shutdown process should be triggered through a migration (upgrade handler) if possible.